M/MSTM/build_tarballs.jl

```diff
@@ -10,14 +10,32 @@ sources = [
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/MSTM/code
-mpifort -O2 mpidefs-parallel.f90 mstm-intrinsics.f90 mstm-v4.0.f90 -o "${bindir}/mstm${exeext}"
+if [[ "$target" == x86_64-w64-mingw32 ]]; then
+    cd $WORKSPACE/destdir/include
+    cp $WORKSPACE/destdir/src/mpi.f90 .
+    gfortran -c -DWIN64 -DINT_PTR_KIND=8 -fno-range-check mpi.f90
```

Comment:
I just realised you're polluting the prefix with files not related to your package.  This isn't good.  Are these modules needed only at buildtime?  If so, then move them to the source directory.

Reply:
I need the MPI headers and libraries only for building the program.

Reply:
Extra files are now removed after compilation.